screwdriver.yaml

```diff
@@ -0,0 +1,44 @@
+---
+workflow:
+    - beta
+
+shared:
+    image: node:6
+
+jobs:
+    main:
+        steps:
+            - check_build_requirements: |
+                which curl
```

Comment:
It's to guarantee that if someone mucks with the image that it'll fail. The `main` job doesn't really do much, since we don't check for anything in a PR build.